src/gatling/simulations/uk/gov/hmcts/jui/sscs/simulations/JUI_SSCSCaseJourney.scala

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.jui.sscs.simulations
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import uk.gov.hmcts.jui.sscs.scenario.utils._
+import uk.gov.hmcts.jui.sscs.scenario._
+import scala.concurrent.duration._
+
+class JUI_SSCSCaseJourney extends Simulation {
+
+ val JUIBaseUrl = scala.util.Properties.envOrElse("URL_TO_TEST", Environment.URL_TO_TEST).toLowerCase()
+
+
+  val httpSSCSProtocol = Environment.HttpSSCSProtocol
+    .baseUrl(JUIBaseUrl)
+    .shareConnections
+    //.proxy(Proxy("proxyout.reform.hmcts.net", 8080))
+
+  val JUISSCSSCN = scenario("SCN_JUI_SSCSJourney")
+    .exec(
+      Logout.logout,
+      Browse.landingLoginPage,
+      JUILogin.submitLogin,
+      JUICases.pickRandomCase,
+      JUIQuestion.sendQuestion,
+      JUIDecision.submitDecision,
+      Logout.logout
+    )
+
+  setUp(JUISSCSSCN.inject(atOnceUsers(1))).protocols(httpSSCSProtocol)
```

Comment:
Why httpSSCSprotocol variable in JUI?

Reply:
That's correct. This nightly pipeline is for SSCS journey process via JUI so I have defined this variable to pick value from Environment.scala object.